test_util/azure.py

```diff
@@ -35,77 +35,123 @@ def validate_hostname_prefix(prefix):
     assert re.match('^[a-z][a-z0-9-]{1,61}[a-z0-9]$', prefix), 'Invalid DNS prefix: {}'.format(prefix)
 
 
+def check_json_object(obj):
```

Comment:
Yeah - this code is intentionally dynamic - in go you'd be using reflection to populate templates.